samcli/lib/observability/xray_traces/xray_event_mappers.py

```diff
@@ -24,8 +24,9 @@ class XRayTraceConsoleMapper(ObservabilityEventMapper[XRayTraceEvent]):
     def map(self, event: XRayTraceEvent) -> XRayTraceEvent:
         formatted_segments = self.format_segments(event.segments)
         iso_formatted_timestamp = datetime.fromtimestamp(event.timestamp).isoformat()
+        revision_info = f"[revision {event.revision}] " if event.revision else ""
         mapped_message = (
-            f"\nXRay Event [revision {event.revision}] at ({iso_formatted_timestamp}) with \
+            f"\nXRay Event {revision_info}at ({iso_formatted_timestamp}) with \
```

Comment:
```suggestion
            f"\nXRay Event {revision_info} at ({iso_formatted_timestamp}) with \
```

Reply:
This was meant to be this way, otherwise when no revision info there will be two spaces between "Event" and "at". @mndeveci

Reply:
Oh got it!